portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,232 @@
+            <div class="slide">
+              <img src="/images/forza/chiron.png" alt="Bugatti Chiron racing down the highway" style="width: 100%">
+            </div>
+            <div class="slide">
+              <img src="/images/forza/ford-gt-night.png" alt="Ford GT at Night" style="width: 100%">
+            </div>
+            <div class="slide">
+              <img src="/images/forza/jeep.png" alt="Jeep driving along the shoreline" style="width: 100%">
+            </div>
+            <div class="slide">
+              <img src="/images/forza/lambo-night.jpg" alt="Lamborghini at Night" style="width: 100%">
+            </div>
+            <div class="slide">
+              <img src="/images/forza/rx7-winter.jpg" alt="Custom RX-7 racing in Winter" style="width: 100%">
+            </div>
+            <div class="slide">
+              <img src="/images/forza/porsche-town.jpg" alt="Porsche Cayman GT4 in Edinburgh" style="width: 100%">
+            </div>
+            <a class="previous-button" onClick="changeSlide(-1)">❮</a>
+            <a class="next-button" onClick="changeSlide(+1)">❯</a>
+
+            <div class="slide-caption">
+              <p id="caption"></p>
+            </div>
+
+            <div class="slideshow-row">
+              <div class="slideshow-column">
+                <img class="slide-demo" src="/images/forza/chiron.png" alt="Bugatti Chiron racing down the highway" style="width: 100%"
+                  onclick="currentSlide(1)">
+              </div>
+              <div class="slideshow-column">
+                <img class="slide-demo" src="/images/forza/ford-gt-night.png" alt="Ford GT at Night" style="width: 100%"
+                  onclick="currentSlide(2)">
+              </div>
+              <div class="slideshow-column">
+                <img class="slide-demo" src="/images/forza/jeep.png" alt="Jeep driving along the shoreline" style="width: 100%"
+                  onclick="currentSlide(3)">
+              </div>
+              <div class="slideshow-column">
+                <img class="slide-demo" src="/images/forza/lambo-night.jpg" alt="Lamborghini at Night" style="width: 100%"
+                  onclick="currentSlide(4)">
+              </div>
+              <div class="slideshow-column">
+                <img class="slide-demo" src="/images/forza/rx7-winter.jpg" alt="Custom RX-7 racing in Winter" style="width: 100%"
+                  onclick="currentSlide(5)">
+              </div>
+              <div class="slideshow-column">
+                <img class=" slide-demo" src="/images/forza/porsche-town.jpg" alt="Porsche Cayman GT4 in Edinburgh" style="width: 100%"
+                  onclick="currentSlide(6)">
+              </div>
+            </div>
+          </div>
+        </div>
+        <p>Photos courtesy of <a href ="https://pz.reddit.com/r/forza/#pz" >r/forza Redditors</a>.</p>
+      </div>
+    
+      <!-- These are blog posts on of my few "unique talents": speedcubing -->
```

Comment:
No need to be so specific here. You can leave out the speedcubing part since the code will already make that obvious.